templates/base/agl-simple.jinja2

```diff
@@ -1,4 +1,4 @@
-{%- extends 'base/agl-base.jinja2' %}
+{%- extends 'base/agl-callback.jinja2' %}
 {%- block deploy -%}
```

Comment:
Not too sure about this.
I think it would be better to use to 'include' agl-callback.jinja2 in agl-base.jinja is callback_name is defined